libs/designsystem/src/lib/components/chart/chart.component.ts

```diff
@@ -23,10 +29,11 @@ import { ChartDataset, ChartHighlightedElements, ChartType } from './chart.types
   providers: [ChartJSService],
 })
 export class ChartComponent implements AfterViewInit, OnChanges {
-  @Input() type: ChartType = 'column';
+  @Input() type: ChartType = ChartTypes.column;
   @Input() data: ChartDataset[] | number[];
   @Input() dataLabels?: string[] | string[][];
   @Input() customOptions?: ChartOptions;
+  @Input() datalabelOptions?: datalabelOptions;
```

Comment:
I'm not entirely sold on the name `dataLabelOptions` - but I got some follow up questions that are easier to clarify irl. I will come by tomorrow 😄